x/docs/md/cli/linux.md

```diff
@@ -68,5 +70,20 @@ exercism configure --dir=~/some/other/place
 ### Continue
 You can now continue by [choosing a language](http://exercism.io/languages).
 
+### Uninstalling the Exercism CLI
+
+Remove the files from the `~/bin` directory with the following command
+
+```bash
+rm -rf ~/bin/exercism
+```
+
+You can also remove the `~/bin` directory from your `PATH` only if you have NOT
+installed anything else in that directory. You can do that by opening up your
+`~/.bashrc` file with your favorite editor and remove the `bin` directory that you
```

Comment:
The cool kids do use zsh, but are only just as cool as the cool kids that use bash or fish or sh or csh. :stuck_out_tongue_winking_eye: 

I think the suggestion of removing the ~/bin folder is not appropriate to hint at, since the hint will happen if upon deleting the exercism file/link the folder is empty.  And it won't hurt if the folder exists in that state.